skimage/morphology/tests/test_rolling_ball.py

```diff
@@ -0,0 +1,39 @@
+"""
+Tests for Rolling Ball Filter
+(skimage.morphology.rolling_ball)
+
+Author: Sebastian Wallkötter
+"""
+
+import numpy as np
+
+from skimage import data
+from skimage.morphology import rolling_ball
+from skimage.util import invert
+
+from skimage._shared.testing import assert_equal, fetch
+from skimage._shared import testing
+
+
+class TestRollingBall(object):
```

Comment:
The class is not needed — see the other testing files for examples.

Reply:
I think there is a mix of having classes or not having classes, e.g., `test_selem.py` or `test_grey.py` use classes, whereas `test_flood_fill.py` uses functions directly.

I only ever use pytest superficially; is there any advantage to using one over the other?

Reply:
We use test functions, usually.  Classes are probably a remainder of older parts of the test suite.

Reply:
Okay, I'll update it.